code/src/components/App.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import Form from './Form';
```

Comment:
Next time one line of empty space between outer and local dependencies would be great :)